src/plone/api/portal.py

```diff
@@ -360,3 +360,36 @@ def set_registry_record(name=None, value=None, interface=None):
         get_registry_record(name)
 
         registry[name] = value
+
+
+def get_default_language():
+    """Return the default language.
+
+    :returns: language identifier
+    :rtype: string
+    :Example: :ref:`portal_get_default_language_example`
+    """
+    try:
+        from Products.CMFPlone.interfaces import ILanguageSchema
```

Comment:
When would importing `ILanguageSchema` raise an import error? Is that a difference between plone versions?